chia/seeder/dns_server.py

```diff
@@ -395,38 +404,68 @@ async def stop(self) -> None:
     async def periodically_get_reliable_peers(self) -> None:
         sleep_interval = 0
         while not self.shutdown_event.is_set() and self.crawl_store is not None:
-            try:
-                new_reliable_peers = await self.crawl_store.get_good_peers()
-            except Exception as e:
-                log.error(f"Error loading reliable peers from database: {e}. Traceback: {traceback.format_exc()}.")
-                continue
```

Comment:
the old behavior here looks problematic. we continue the loop without sleeping. This is fixed by your patch though